packages/react/src/DispatchCell.tsx

```diff
@@ -31,9 +31,19 @@ import { withJsonFormsDispatchCellProps } from './JsonFormsContext';
 /**
  * Dispatch renderer component for cells.
  */
-export const Dispatch = ({uischema, schema, rootSchema, path, cells, id, enabled, renderers}:DispatchCellProps) => {
-  const cell = useMemo(() => maxBy(cells, r => r.tester(uischema, schema, rootSchema)), [cells, uischema, schema]);
-  if (cell === undefined || cell.tester(uischema, schema, rootSchema) === -1) {
+export const Dispatch = ({uischema, schema, rootSchema, path, cells, id, enabled, renderers, config }: DispatchCellProps) => {
+  const testerContext = useMemo(
+    () => ({
+      rootSchema: rootSchema,
+      config: config
+    }),
+    [rootSchema, config]
+  );
+  const cell = useMemo(
+    () => maxBy(cells, r => r.tester(uischema, schema, testerContext)),
+    [cells, uischema, schema, testerContext]
+  );
+  if (cell === undefined || cell.tester(uischema, schema, testerContext) === -1) {
```

Comment:
Personally I would have done one `useMemo` as the standalone `testerContext` `useMemo` is not really necessary but that's just a minor issue.